package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -292,6 +293,25 @@ dhcp_host_add() {
 	xappend "--dhcp-host=$macs${networkid:+,net:$networkid}${broadcast:+,set:needs-broadcast}${tag:+,set:$tag}${ip:+,$ip${hostid:+,[::$hostid]}}${name:+,$name}${leasetime:+,$leasetime}"
 }
 
+dhcp_this_host_add() {
+	# TODO: case-in do/not short-host or FQDN; UCI already intended ...
+	local ifdashname="$1"
+	local do_enable="$2"
+	local routerstub routername
+	
+	ifdashname="${ifdashname//./-}"
+	routerstub="$( cat /sys/class/net/br-lan/address )"
```

Comment:
Oops. That wasn't the best choice either. If no host name (what?), then just MD5 /etc/os-release.